### PR TITLE
Homework 3

### PR DESCRIPTION
**Pull request by OlexiyVasylenkoDev**

### CODE REVIEW

File: homework3/homework3.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    {
+        'count': fields.Int(
+            load_default=10,
+            validate=[validate.Range(min=1, max=1000)]
+        ),
+    },
+    location="query"
+)
+def generate_students(count):
+    with open('students.csv', 'w') as my_csv:
```
Comment (mikolaz27) [inline homework3/homework3.py]: bad idea,
try another approach:
1. Generate data and keep it in variable
2. Same data save to file 
3. Same data show on frontend

File: homework3/homework3.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+@app.route("/bitcoin")
+@use_kwargs(
+    {
+        'currency': fields.Str(
+            load_default='USD',
+        ),
+        'count': fields.Int(
+            load_default=1,
+            validate=[validate.Range(min=1)]
```
Comment (mikolaz27) [inline homework3/homework3.py]: set max value to avoid problems with user input

File: homework3/homework3.py [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        )
+    },
+    location="query"
+)
+def get_bitcoin_value(currency, count):
+    currency = currency.upper()
+    url = 'https://bitpay.com/api/rates'
```
Comment (mikolaz27) [inline homework3/homework3.py]: check documentation more deeply
here should be something like that https://bitpay.com/api/rates/<currency>

File: homework3/homework3.py [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+    result_for_symbols = requests.get(url_for_symbols, {})
+    result_for_symbols = result_for_symbols.json()
+
+    for row in result:
+        if currency == row['code']:
+            currency_rate = int(row['rate'])
+    for row in result_for_symbols['data']:
+        if currency == row['code']:
+            currency_symbol = row['symbol']
+    if count != 1:
```
Comment (mikolaz27) [inline homework3/homework3.py]: better to set count as None as default value and here just check if count is None

**Review by mikolaz27 — CHANGES_REQUESTED** [review]

File: homework3/homework3.py (hunk 3 above)
Comment (OlexiyVasylenkoDev) [reply homework3/homework3.py]: Didn't get, what's wrong here?
Probably, you've sent the wrong link, because right now its exactly the same as in my code)

**Pull request merged**